pkg/scheduler/scheduler.go

```diff
@@ -199,6 +177,37 @@ func New(config Config) *KubernetesScheduler {
 	return k
 }
 
+// isCompatible returns true if the specified offer is compatible with the scheduler's executor.
+// Offers are compatible unless the name matches and the group doesn't.
+func (k *KubernetesScheduler) isOfferCompatible(o *mesos.Offer) bool {
+	for _, eid := range o.GetExecutorIds() {
+		executorIdString := eid.GetValue()
+		executorID, err := uid.Parse(executorIdString)
+		if err != nil {
```

Comment:
offers are expected to arrive with executor IDs that are unrecognized by this framework. this logic would end up rejecting all offers from slaves that are running an executor whose ID is unparseable by us which is probably nearly all of them.

what we want to avoid the case where we validate an offer for resources on a slave that is running an executor launched by this framework (or some prior incarnation of it) with a different configuration (that's why we hash the executor info).